requirements.txt

```diff
@@ -1,6 +1,9 @@
 importlib_metadata
 jsonschema
-pytest>=5.0.0
+pytest>=5.0.0,<6.0.0
```

Comment:
`pytest >= 6` gives errors during the installation. I guess it's a problem with pytest's requirements and the new resolver.